### ISSUE TITLE
Vulnerabilities in dependencies of neo4j official container image

### ISSUE THREAD

**Issue by joancafom** [neo4j/neo4j#13223]
I have recently analysed the latest release of the [5.9.0 docker image](https://hub.docker.com/layers/library/neo4j/5.9.0-community/images/sha256-fa8f71d6091bc76419f571acd04d30fd654642c7f834e5d671f637303611425d?context=explore) that is published in your DockerHub image registry with [Trivy](https://github.com/aquasecurity/trivy) (a vulnerability and security scanner). The scanner reports a couple of vulnerabilities in various dependencies:

```bash
$ trivy image --vuln-type library neo4j:5.9.0-community
...
Java (jar)

Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 2, HIGH: 0, CRITICAL: 0)

┌─────────────────────────────────────────────────────────┬────────────────┬──────────┬───────────────────┬───────────────┬─────────────────────────────────────────────────────────┐
│                         Library                         │ Vulnerability  │ Severity │ Installed Version │ Fixed Version │                          Title                          │
├─────────────────────────────────────────────────────────┼────────────────┼──────────┼───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ com.google.guava:guava (apoc-5.9.0-core.jar)            │ CVE-2023-2976  │ MEDIUM   │ 31.0.1-jre        │ 32.0.0        │ Guava vulnerable to insecure use of temporary directory │
│                                                         │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-2976               │
├─────────────────────────────────────────────────────────┼────────────────┤          ├───────────────────┼───────────────┼─────────────────────────────────────────────────────────┤
│ io.netty:netty-handler (netty-handler-4.1.92.Final.jar) │ CVE-2023-34462 │          │ 4.1.92.Final      │ 4.1.94.Final  │ netty-handler SniHandler 16MB allocation                │
│                                                         │                │          │                   │               │ https://avd.aquasec.com/nvd/cve-2023-34462              │
└─────────────────────────────────────────────────────────┴────────────────┴──────────┴───────────────────┴───────────────┴─────────────────────────────────────────────────────────┘
```

Do you have any information/statement about these CVEs related to your product? Is your software actually affected by them? If so, do you plan to update the dependencies to a fixed version?

**Neo4j Version:** 5.9.0
**Operating System:** Ubuntu 16.04
**API:** Docker

### Steps to reproduce
1. Analyse the image with Trivy: `trivy image --vuln-type library neo4j:5.9.0-community`

### Expected behavior
Vulnerabilities in dependencies that affect the product should be addressed.

**Comment by joancafom** [comment]
Hi again, just to mention that a new vulnerability is now reported as well:

```bash
trivy image --vuln-type library neo4j:5.9.0-community

Java (jar)

Total: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 1, HIGH: 1, CRITICAL: 0)

┌──────────────────────────────────────────────────────────────┬─────────────────────┬──────────┬───────────────────┬───────────────┬───────────────────────────────────────────────────┐
│                           Library                            │    Vulnerability    │ Severity │ Installed Version │ Fixed Version │                       Title                       │
├──────────────────────────────────────────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼───────────────────────────────────────────────────┤
(...)
├──────────────────────────────────────────────────────────────┼─────────────────────┼──────────┼───────────────────┼───────────────┼───────────────────────────────────────────────────┤
│ org.eclipse.jetty:jetty-xml (jetty-xml-9.4.51.v20230217.jar) │ GHSA-58qw-p7qm-5rvh │ LOW      │ 9.4.51.v20230217  │               │ Eclipse Jetty XmlParser allows arbitrary DOCTYPE  │
│                                                              │                     │          │                   │               │ declarations                                      │
│                                                              │                     │          │                   │               │ https://github.com/advisories/GHSA-58qw-p7qm-5rvh │
└──────────────────────────────────────────────────────────────┴─────────────────────┴──────────┴───────────────────┴───────────────┴───────────────────────────────────────────────────┘
```

**Comment by gjmwoods** [comment]
Hi @joancafom, many thanks for raising this and apologies for our delay in responding.

We are also scanning the product as you have for vulnerabilities which we typically fix in the latest minor version unless a critical vulnerability has been discovered which will lead to a faster patch release.

The ones you've listed here have all been resolved in 5.11 so I will close this issue. Many thanks again.
